addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -154,13 +135,33 @@
                                 class="oe_stat_button">
                                 <field name="bounced_ratio" string="Bounced" widget="percentpie"/>
                             </button>
+                            <button name="action_view_traces_sent"
+                                icon="fa-paper-plane"
```

Comment:
Linting: Globally here some properties could be set on same line to reduce file length.